index.html

```diff
@@ -167,8 +260,10 @@ <h2 class="color-secondary">About me</h2>
     </div>
     </section>
     <section class="contact">
+        <div class="email-header">
+            <h4 id="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+        </div>
         <div class="email">
-            <h4 id="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h4>
             <form action="https://formspree.io/f/xpzewrow" method="post">
```

Comment:
Kindly fix the little alignment issue on the left of the Get in Touch button with the rest of the input areas. 👌
<img width="853" alt="Screenshot 2023-01-20 at 9 13 23 AM" src="https://user-images.githubusercontent.com/25887253/213629874-e99e7cb6-231d-4e7b-818e-4d40fd368a29.png">